landoapi/treestatus.py

```diff
@@ -29,7 +29,7 @@ def __init__(self, *, url=None, session=None):
         self.url = self.url if self.url[-1] == "/" else self.url + "/"
         self.session = session or self.create_session()
 
-    def is_open(self, tree):
+    def is_open(self, tree) -> bool:
```

Comment:
nit: Why not also add type hints for the arguments as well, here and below -- since you're here already?

Reply:
Good point, I'll do that now.